config/config.go

```diff
@@ -36,8 +35,7 @@ func Get() (cfg *Config, err error) {
 		BindAddr:                   ":20200",
 		ZebedeeURL:                 "http://localhost:8082",
 		RendererURL:                "http://localhost:20010",
-		FilterAPIURL:               "http://localhost:22100",
-		DatasetAPIURL:              "http://localhost:22000",
+		APIRouterURL:               "http://localhost:23200/v1",
```

Comment:
the code changes look good to me, but we will need to address the API versioning issue before merging it.